src/index.js

```diff
@@ -0,0 +1,97 @@
+require('./css/styles.css');
+
+const description = document.querySelector('.main');
+const iconTag = document.querySelector('#iconWeather');
+const mainTemp = document.querySelector('.mainTemp');
+const maxTemp = document.querySelector('.maxTemp');
+const minTemp = document.querySelector('.minTemp');
+const windSpeed = document.querySelector('.windSpeed');
+const cityElement = document.querySelector('.cityName');
+const cityInput = document.querySelector('#cityInput');
+const submit = document.querySelector('#submit');
+const locationBtn = document.querySelector('#getLocationBtn');
+const celOrFah = document.querySelector('#checkBox');
+let userCity = 'Baghdad';
+
+function fillElements({
+  name,
+  weather,
+  main,
+  wind,
+}) {
+  cityElement.textContent = name;
+  description.textContent = weather[0].description;
+  const { icon } = weather[0];
+  iconTag.src = `https://openweathermap.org/img/wn/${icon}@2x.png`;
+  mainTemp.innerHTML = `${main.temp}&#8451;`;
+  maxTemp.innerHTML = `${main.temp_max}&#8451;`;
+  minTemp.innerHTML = `${main.temp_min}&#8451;`;
+  windSpeed.textContent = `Wind Speed: ${wind.speed}`;
+}
+
+async function fetchApi(cityName) {
+  try {
+    const result = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${process.env.API_KEY}&units=metric`,
+    );
+    const data = await result.json();
+    fillElements(data);
+  } catch (err) {
+    cityElement.textContent = err;
+  }
+}
+
+async function fetchGeoLocationApi(lat, lon) {
+  try {
+    const result = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${process.env.API_KEY}&units=metric`,
+    );
+    const data = await result.json();
+    fillElements(data);
+  } catch (err) {
+    cityElement.textContent = err;
+  }
+}
+
+function getUserInput() {
```

Comment:
usage of arrow functions instead of the `function` keyword, kindly do same for all other functions declare without the use of arrow functions below